Examples_Nuget/ArrayShareExample/ArrayShareExample.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="TileDB.Cloud" Version="0.2.2" />
+    <PackageReference Include="TileDB.Cloud" Version="0.2.3" />
```

Comment:
Version 0.2.2 was accidentally referencing the unpublished `TileDB.Rest` package. I saw it was unlisted from NuGet.org, I also marked it as deprecated to make the problem more visible.